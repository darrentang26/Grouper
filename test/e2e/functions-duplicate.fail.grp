let Int one = 
    let [Int] -> Int plusOne = lambda (Int x) x + 1 end
    in plusOne 0
and Int two = 
    let [Int] -> Int plusOne = lambda (Int x) x + 1 end
    in plusOne 1
in print 1 + 2