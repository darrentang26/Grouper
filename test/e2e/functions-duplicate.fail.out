Fatal error: exception Lifting.Failure("function plusOne is previously defined")
